sources/packages/web/src/components/generic/ModalDialogBase.vue

```diff
@@ -14,17 +14,17 @@
     >
       <v-card-title>
         <slot name="header">
-          <h2 v-if="title" class="category-header-large primary-color">
+          <h2 v-if="title" class="category-header-large primary-color mt-3">
             {{ title }}
           </h2>
         </slot>
       </v-card-title>
-      <v-divider class="border-opacity-100" :thickness="2" inset />
-      <v-card-text class="pt-0">
+      <v-divider class="border-opacity-100 my-0" :thickness="2" inset />
+      <v-card-text class="pt-0 pb-1">
         <div class="pb-2" v-if="subTitle">{{ subTitle }}</div>
         <slot name="content">Please add the modal content here!</slot>
       </v-card-text>
-      <v-divider class="border-opacity-100" :thickness="2" inset />
+      <v-divider class="border-opacity-100 my-0" :thickness="2" inset />
```

Comment:
why this change in generic component?

Reply:
Without adding the `my-0` to the v-dividers, they are adding a default margin which increases a lot of vertical white space between the elements above and below it. `mt-3` was added to the title because there is no top margin otherwise and the component doesn't match Figma in that case.

Reply:
I have the updated the `modal-dialog-base` component reducing the number of changes to the component and I have looked up many pre-existing modal dialogs. They look fine with the changes.